SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -344,6 +341,11 @@ void CDiscAdjSolver::ExtractAdjoint_Solution(CGeometry *geometry, CConfig *confi
   }
   END_SU2_OMP_FOR
 
+  /*--- Residuals and time_n terms are not needed when evaluating multizone cross terms. ---*/
+  if (CrossTerm) return;
```

Comment:
well ... if thought about this way via return as well but went with the smaller change ... i guess you merge-conflicted that away(?)